scripts/test/watch

```diff
@@ -1,12 +0,0 @@
-#!/bin/sh
-
-set -e
-
-filewatcher \
-    -L 5 \
-    -x '**/*.swp' \
-    -x .git \
-    -x build \
-    -x .idea \
-    -- \
-    sh -c 'go test -timeout 10s -v ./${dir} || ( echo; echo; exit 1 )'
```

Comment:
>  It is a dependency that needs to downloaded 

True, but the time it takes to download is insignificant compared to the other steps in the Dockerfile.

> a workflow that doesn't work very well in Go

Why do you say it doesn't work very well? Have you ever tried it?

I've been using it for as long as I've been writing GO and it's always worked perfectly. It may not be obvious from your quick glance, but this actually runs just the unit tests for the package that is being modified, not for the entire repo.